src/ClientData/include/ClientData/CaptureData.h

```diff
@@ -249,6 +249,10 @@ class CaptureData {
       uint64_t min_tick = std::numeric_limits<uint64_t>::min(),
       uint64_t max_tick = std::numeric_limits<uint64_t>::max()) const;
 
+  [[nodiscard]] std::vector<const TimerInfo*> GetAllTimersForScope(
```

Comment:
I would call this `GetTimersForScope`, to distinguish it more from `GetAllScopeTimers`. Yes, it's all for that timer in that time range, but "all" doesn't really add information and having the contrast is better.